go.mod

```diff
@@ -125,3 +127,5 @@ require (
 )
 
 replace github.com/docker/docker => github.com/docker/docker v20.10.3-0.20220224222438-c78f6963a1c0+incompatible
+
+replace github.com/goharbor/acceleration-service v0.0.0 => github.com/imeoer/acceleration-service v0.0.6
```

Comment:
Is this just temporary for development?

Reply:
Thanks for the reply, will upgrade to the release version before PR finishes.